awesome_provider/app/src/provider_model.py

```diff
@@ -1,14 +1,15 @@
 import queryHelper
 
-class provider_moderl:
-    def __init__(self , url, usr, password, dbName):
-        self.query = QueryHelper(url, usr, password, dbName)
+class provider_model:
+    #def __init__(self , url, usr, password, dbName):
+     #   self.query = QueryHelper(url, usr, password, dbName)
 
-    def CheckHealth(self):
+    def CheckHealth():
```

Comment:
IF this is a method that belongs to the class, you have to pass `self`

Reply:
using the method as static method will fix in  later version